docs/developer-docs/integrations/sns/launching/launch-steps.md

```diff
@@ -68,14 +68,70 @@ sns deploy --network "${NETWORK}" --init-config-file "${CONFIG}" --save-to "sns_
 
-Again, anyone who owns an NNS neuron with enough stake can submit this proposal.
+Note that this proposal needs to be submitted by one of the initial SNS neurons. This means that the devs require at least one SNS neuron that they can control with the command line tool quill. They can then use the following command to submit the SNS proposal.
 Of course it is crucial to set the right parameters in this proposal.
-You can also find an example how this command is used in the SNS local testing
-[here](https://github.com/dfinity/sns-testing/blob/main/open_sns_sale.sh#L11-L26).
+
+#### Quill command
+
+```bash
+quill sns \
+   --canister-ids-file ./sns_canister_ids.json \
+   --pem-file "${PEM_FILE}" \
+   make-proposal \
+   --proposal \
+      "(record { \
+         title=\"${REGISTER_DAPP_PROPOSAL_TITLE}\"; \
+         url=\"${REGISTER_DAPP_PROPOSAL_URL}\"; \
+         summary=\"${REGISTER_DAPP_PROPOSAL_SUMMARY}\"; \
+         action=opt variant {RegisterDappCanisters = record {canister_ids=${DAPP_CANISTER_IDS}}}})" \
+   "${DEVELOPER_NEURON_ID}"
 ```
+
+Where the parameters are:
+
+* `sns_canister_ids.json` is a file with the IDs of this SNS’s canisters
+
+* `PEM_FILE`="$(dfx cache show)/../../${IDENTITY_NAME}/identity.pem"
+
+* `IDENTITY_NAME`=`<current-dapp-controller-identity-name>`
+
+* `REGISTER_DAPP_PROPOSAL_TITLE`="Register test dapp"
+
+* `REGISTER_DAPP_PROPOSAL_URL`="https://example.com/"
+
+* `REGISTER_DAPP_PROPOSAL_SUMMARY`="This proposal registers test dapp with SNS"
+
+* `DAPP_CANISTER_IDS`="vec { \
+ principal\"$CID1\"; \
+ principal\"$CID2\"; \
+ principal\"$CID3\" \
+} where we assume your dapp’s canister principals are encoded as strings and set into variables named `CID2`, `CID2`, `CID3`
+
+* `DEVELOPER_NEURON_ID`=`<your-developer-neuron-ID>`
+
+### 6. Proposal #2 (of 3) is passed or rejected
+
+Dapp developers vote with their initial SNS neurons. Depending on the initial neuron distirbution, this can also include other voters that have initial neurons. 
+
+Note that at this stage it is not possible to use the NNS frontend dapp to vote on SNS proposals - the NNS FE dapp will only show SNS neurons and proposals for SNSs that are fully launched. 
+Therefore, there are two options how this proposal can be adopted:
+
+1) The proposal is submitted by a neuron that has at least 3% of the voting power and one waits for the voting period of 4 days to be complete. If no other neurons vote, the proposal will then be adopted after this voting period.
+
+2) The initial neurons are set up in such a way that the majority of the voting power is in neurons that are controlled via quill. In this case, other neurons can vote in this step and the proposal can potentially pass earlier if an absolute majority of the voting power is reached.
+
+initial neuron distribution, this also includes community votes.
```

Comment:
Is this a section title or a fragmented sentence?

Reply:
quickly looking at it I think line 125 is a left-over, but will let Diego chime in